MuxExoPlayer/build.gradle

```diff
@@ -254,6 +275,13 @@ afterEvaluate {
                 artifactId = "MuxExoPlayer_r2_14_1"
                 version = project.ext.versionName
             }
+
+            r2_15_1Release(MavenPublication) {
+                from components.r2_14_1Release
```

Comment:
This needs to be from components.r2_15_1Release.

Reply:
Bumping this ^^^